rpc_jobs/rpc_aio.yml

```diff
@@ -266,7 +266,7 @@
       env.ACTION = "{action}"
       env.SERIES = "{series}"
       timeout(time: 8, unit: 'HOURS'){{
-        node() {{
+        common.shared_slave() {{
```

Comment:
Good point, I'm going to have a look at implicitly loaded libraries again to see if this can be made to work.